src/main/java/bit/javaoop/CEO.java

```diff
@@ -0,0 +1,26 @@
+package bit.javaoop;
+
+import java.util.ArrayList;
+
+public class CEO {
+    private ArrayList<AbstractEmployee> listOfEmployees = new ArrayList<>();
+    private String name;
+
+    public CEO(String name) {
+        this.name = name;
+    }
+
+
+    public String getName() {
+        return name;
+    }
+
+    public ArrayList<AbstractEmployee> getListOfEmployees() {
+        return listOfEmployees;
```

Comment:
- W javie return zapewnia nam, że zwracamy kopie ale jedynie bezpośrednio tego co zwracamy czyli jeśli mielibyśmy getter na typ prymitywny to zostanie z tego gettera zwrócona kopia naszej zmiennej ale w przypadku obiektów, zwracana jest kopia referencji a nie kopia obiektu! Co oznacza, że referencji nam nikt nie będzie mógł zmienić bo ma kopie ale już obiekt, na który wskazuje ta referencja jak najbardziej ( z wyjątkiem obiektów niemutowalnych np. stringa ).
- Do czego zmierzam zwracając tak bezpośrednio referencje do listy, ktoś może nam ją popsuć dużo lepiej jest zwrócić kopię tutaj np. w ten sposób return new ArrayList<>(listOfEmployees); załatwi nam sprawę:)